main.go

```diff
@@ -53,7 +54,8 @@ func main() {
 		metricsPath = kingpin.Flag("web.telemetry-path",
 			"Path under which to expose metrics.").
 			Default("/metrics").String()
-		esURI = kingpin.Flag("es.uri",
+		tlsConfigFile = kingpin.Flag("web.config", "Path to config yaml file that can enable TLS").Default("").String()
```

Comment:
This should be handled by the kingpinflag module from the exporter-toolkit.
`webflag "github.com/prometheus/exporter-toolkit/web/kingpinflag"`

```suggestion
		webConfig = webflag.AddFlags(kingpin.CommandLine)
```